docs/docs/delegators/delegator-faq.md

```diff
@@ -5,7 +5,7 @@ order: 4
 
 ## What is a delegator?
 
-People that cannot or do not want to operate [validator nodes](/validators/overview.mdx) can still participate in the staking process as delegators. Indeed, validators are not chosen based on their self-delegated stake but based on their total stake, which is the sum of their self-delegated stake and of the stake that is delegated to them. This is an important property, as it makes delegators a safeguard against validators that exhibit bad behavior. If a validator misbehaves, their delegators will move their Atoms away from them, thereby reducing their stake. Eventually, if a validator's stake falls under the top 180 addresses with highest stake, they will exit the validator set.
+People that cannot or do not want to operate [validator nodes](/validators/overview) can still participate in the staking process as delegators. Indeed, validators are not chosen based on their self-delegated stake but based on their total stake, which is the sum of their self-delegated stake and of the stake that is delegated to them. This is an important property, as it makes delegators a safeguard against validators that exhibit bad behavior. If a validator misbehaves, their delegators will move their Atoms away from them, thereby reducing their stake. Eventually, if a validator's stake falls under the top 180 addresses with highest stake, they will exit the validator set.
```

Comment:
broken link

Reply:
Oh, you're trying to navigate this on your local machine without the docs server runnning. You're just looking at the files from vscode?
The link is not broken, it just cannot navigate you to the correct resource without the docs server running.